### ISSUE TITLE
Better Invites crashing discord when an invite link is displayed

### ISSUE THREAD

**Issue by zany130** [HypedDomi/BetterDiscordStuff#24]
### Describe the bug

Invite links in chanels crash discord as of the latest discord update	

### How can I recreate this bug

1.update discord and better discord
2.open a channel with a invite link posted in it 
3. see crash

**Comment by XxUnkn0wnxX** [comment]
ya i'm getting this as well

**Comment by HypedDomi** [comment]
I dont have much time for BD stuff atm but I’ll try my best to fix it

**Comment by rocord01** [comment]
I am having this same issue.

**Comment by THEGAMER20081** [comment]
same issue its happening with invalid invites

**Comment by vracton** [comment]
I face this as well, the error given is a React one; **Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined.**

**Comment by ShizCalev** [comment]
Not sure if it might help, but a couple other plugins had to update from using getByPrototypeFields to using the new BdApi to fix crashing with the latest update.
https://github.com/Strencher/BetterDiscordStuff/issues/334#issuecomment-1455243589

Figure it could possibly lead you in the right direction.

**Comment by thealzhe** [comment]
## BetterInvites crashing my client.
```
BetterDiscord 1.9.2
Stable 199933 (db1e138)
Host 1.0.9013 (32266)
```
## This error is dropped when URL its prompt to reload our Discord Client.
```TypeScript
575bf98fd12b6dadfa04.js:241 Error: Minified React error #130; visit https://reactjs.org/docs/error-decoder.html?invariant=130&args[]=undefined&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at kc (ad1fde4a477ddef962ae.js:1838:511)
    at d (ad1fde4a477ddef962ae.js:1773:772)
    at E (ad1fde4a477ddef962ae.js:1775:165)
    at e (ad1fde4a477ddef962ae.js:1777:306)
    at Os (ad1fde4a477ddef962ae.js:1789:192)
    at gu (ad1fde4a477ddef962ae.js:1834:962)
    at Ic (ad1fde4a477ddef962ae.js:1826:89)
    at mc (ad1fde4a477ddef962ae.js:1826:17)
    at hc (ad1fde4a477ddef962ae.js:1825:867)
    at oc (ad1fde4a477ddef962ae.js:1822:642)
    at ic (ad1fde4a477ddef962ae.js:1821:184)
    at v (ad1fde4a477ddef962ae.js:1943:904)
    at C (ad1fde4a477ddef962ae.js:1944:267)
    at A.<computed> (575bf98fd12b6dadfa04.js:76:1)
    at k (575bf98fd12b6dadfa04.js:75:764)
    at MessagePort.j (575bf98fd12b6dadfa04.js:75:823)
```

**Comment by HolyPeanut** [comment]
Hey, getting the same bug and the same error log, disabling "show user who invited you" and "show guild verif. level" fixes it

**Comment by HypedDomi** [comment]
Fixed with [`9c0421b`](https://github.com/HypedDomi/BetterDiscordStuff/commit/9c0421bf53ecd914348fbbf079b1e657ec16c47e)
Thanks to @ImAFrogOwO
